mcsema/CFG/CFG.cpp

```diff
@@ -865,6 +875,24 @@ NativeModule *ReadProtoBuf(const std::string &file_name,
     auto func = const_cast<NativeFunction *>(
         module->ea_to_func[static_cast<uint64_t>(cfg_func.ea())]);
 
+    // Extract the stack variables associated with the function
+    for (const auto &stack_var : cfg_func.stack_vars()) {
+      auto nat_sv = new NativeStackVariable;
+
+      nat_sv->offset = stack_var.sp_offset();
+      nat_sv->size = stack_var.size();
+      nat_sv->ea = 0;
+      nat_sv->name = stack_var.name();
+      nat_sv->lifted_name = LiftedStackVariableName(stack_var);
+      nat_sv->llvm_var = nullptr;
+      func->stack_vars.push_back(nat_sv);
+
+      for(auto ref_ea : stack_var.ref_eas()){
+        nat_sv->refs[ref_ea.inst_addr()] = ref_ea.offset();
```

Comment:
Can you rename `inst_addr` in the CFG.proto to `inst_ea` to have the same naming as other `ea`-related stuff.